code/datums/gamemode/vampire_gamemode.dm

```diff
@@ -1,4 +1,122 @@
 /datum/gamemode/vampire
 	minimum_player_count = 10
 	name = "Vampire"
-	factions_allowed = list(/datum/faction/vampire)
\ No newline at end of file
+	factions_allowed = list(/datum/faction/vampire)
+
+// -- Vampire powers
+
+/datum/power/vampire
+	name = "Power"
+	desc = "Placeholder"
+	helptext = "No help text for vamps."
+	isVerb = 0 	// Is it an active power, or passive?
+	verbpath = null // Vampires have no verbs
+
+	var/blood_threeshold = 0
+	var/spell_path
+	var/id
+
+/datum/power/vampire/proc/Give(var/datum/role/vampire/V)
+	if (!istype(V))
+		message_admins("Error: trying to give a vampire power to a non-vampire.")
+
+	if (spell_path)
+		var/spell/S = new spell_path
+		V.antag.current.add_spell(S)
+
+	V.powers |= id
+
+// -- List of all vampire spells
+
+/* Tier 0 : roundstart */
+/datum/power/vampire/rejuvenate
+	blood_threeshold = 0
+	id = VAMP_REJUV
+	spell_path = /spell/rejuvenate
+
+/datum/power/vampire/glare
+	blood_threeshold = 0
+	id = VAMP_GLARE
+	spell_path = /spell/aoe_turf/glare
+
+/datum/power/vampire/hypnotise
+	blood_threeshold = 0
+	id = VAMP_HYPNO
+	spell_path = /spell/targeted/hypnotise
+
+/* Tier 1 */
+/datum/power/vampire/shape
+	blood_threeshold = 100
+	id = VAMP_SHAPE
+	spell_path = /spell/shapeshift
+
+/datum/power/vampire/vision
+	blood_threeshold = 100
+	id = VAMP_VISION
+	spell_path = null // TODO
+
+/* Tier 2 */
+/datum/power/vampire/disaese
```

Comment:
>disaese

je pense c'est peut-etre une erreur

Reply:
There he goes again. Look, everyone! He posted it once again! Isn't he just the funniest guy around?!